istioctl/cmd/auth_test.go

```diff
@@ -249,6 +249,20 @@ func TestAuthConvert(t *testing.T) {
 			configMapFile: "testdata/auth/converter/istio-configmap.yaml",
 			golden:        "testdata/auth/converter/cluster-rbac-config-on-with-exclusion.golden.yaml",
 		},
+		{
+			name: "Multiple access rules with one subject",
+			rbacV1alpha1Files: []string{"testdata/auth/converter/multiple-access-rules.yaml",
```

Comment:
Add a newline, see how other lines are intended, here and below.

Reply:
Done.